app/models/miq_queue.rb

```diff
@@ -153,6 +153,19 @@ def self.put(options)
     msg
   end
 
+  # Execute a job on all servers.  Filterable by zone
+  #
+  def self.broadcast(options)
```

Comment:
Don't use an options hash for a method that can only take one possible option

Reply:
It's options that will be forwarded to put.